website/client/components/chat/chatMessages.vue

```diff
@@ -24,11 +24,12 @@
         .message-hidden(v-if='msg.flagCount > 1 && user.contributor.admin') Message hidden
         .card-block
             h3.leader(
-              :class='userLevelStyle(cachedProfileData[msg.uuid])'
+              :class='userLevelStyle(msg)'
```

Comment:
why was this changed?

Reply:
In the above PR, @Alys requested that instead of using the cached profile for contributor info, that I use the information stored in the chat message itself.

Reply:
you're right, sorry!